package.json

```diff
@@ -8,7 +8,18 @@
     "ember-data-addon"
   ],
   "license": "MIT",
-  "author": "",
+  "author": "David J. Hamilton <dhamilton@linkedin.com>",
+  "contributors": [
```

Comment:
I'm unfamiliar with the limitations of this field, but are we able to add github / npm emails here?

Reply:
Can be either of these formats:

```js
[
  { name: 'Robert Jackson', email: 'me@rwjblue.com', url: 'https://www.rwjblue.com' }
]
```

Or:

```js
[
  'Robert Jackson <me@rwjblue.com> (https://www.rwjblue.com)'
]
```

Any of the three parts can be left off.

Documentation here: https://docs.npmjs.com/files/package.json#people-fields-author-contributors

Reply:
"Thomas Wang <wangxgwxg@gmail.com> (https://twitter.com/xinganwang)"

Thanks :)

Reply:
'Steve Calvert <steve.calvert@gmail.com> (https://www.twitter.com/scalvert)'

Reply:
'Danail Nachev (http://www.linkedin.com/in/danailnachev)'

Reply:
"Sang Mercado sang.mercado@gmail.com (http://linkedin.com/in/sangmercado)"

Reply:
"Dinesh Nalagatla dinu.nalagatla@gmail (https://www.linkedin.com/in/dineshnalagatla)"
Thank you

Reply:
> Hopefully, it's OK to skip on the email :) Not that all mail list doesn't have it already.

It's 💯 .  The entire reason I didn't preemptively include emails is I that I don't want to publish peoples' contact info without their explicit preference, even if it's ultimately a pretty futile gesture.